routing/router_test.go

```diff
@@ -104,14 +104,14 @@ func TestRedirectWithSetWebRootKeepsParams(t *testing.T) {
 		t.Fatal(err)
 	}
 	body, _ := ioutil.ReadAll(resp.Body)
-	assert.Equal(t, 200, resp.StatusCode, "Response should not redirect")
+	assert.Equal(t, 500, resp.StatusCode, "Response should not redirect")
```

Comment:
is this even testing anything now? It should result in a successful request, but if we can't write to index.html because we don't have a console/index.html file, then this isn't testing what it was testing before, right?

Reply:
My thought is that this is testing that a redirection is *not* happening (or that's what I understand from the "Response should not redirect" string). 
Specifically, I *think* this is testing that this line is not being hit: https://github.com/kiali/kiali/blob/master/routing/router.go#L28. That line would return a `302` status code. So, my rationale was that as long as we don't get the `302` status code, the test should be green.

I had a similar rationale for `server_test.go`: as long as we get that `500` response, which is the expected within the test, I think it should be OK.

But I think mocking the index.html at runtime is fine. I'll do the changes.